mission_control/navigator_missions/nodes/run_mission

```diff
@@ -11,7 +11,7 @@ import argparse
 @util.cancellableInlineCallbacks
 def main():
     nh, args = yield NodeHandle.from_argv_with_remaining("navigator_mission_runner")
-    available_missions = [mission_name for mission_name in dir(nav_missions) if mission_name[0] != '_']
+    available_missions = [mission_name for mission_name in dir(nav_missions)]
```

Comment:
Not sure why these two changes are here. I didn't make them so they might be from your PR to my fork.